shared/src/common/enums/api/bs-object-path.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum BSObjectApiPath {
+  SPACES_FILE = '/spaces/:id/objects',
```

Comment:
add this path to https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/blob/d19b44ef6f5d0d6743296369369f1f02516ccabc/shared/src/common/enums/api/spaces-api-path.enum.ts